spec/lib/api/2.0/nodes-spec.js

```diff
@@ -857,4 +857,49 @@ describe('2.0 Http.Api.Nodes', function () {
         });
     });
 
+    describe('OBM support', function() {
+
+        var obm = {
+            service: 'noop-obm-service',
+            config: {
+                host: '1.2.3.4',
+                user: 'myuser',
+                password: 'mypass'
+            }
+        };
+
+
+        before(function() {
+            sinon.stub(nodesApi, 'getObmsByNodeId').resolves([]);
+            sinon.stub(nodesApi, 'putObmsByNodeId').resolves([]);
+        });
+
+        after(function() {
+            nodesApi.getObmsByNodeId.restore();
+            nodesApi.putObmsByNodeId.restore();
+        });
+
+        it('should call getObmsByNodeId', function() {
+            return helper.request().get('/api/2.0/nodes/123/obm')
+                .expect('Content-Type', /^application\/json/)
+                .expect(200)
+                .expect(function() {
+                    expect(nodesApi.getObmsByNodeId).to.have.been.calledWith('123');
+                });
+        });
+
+        it('should call putObmsByNodeId', function() {
+            return helper.request().put('/api/2.0/nodes/123/obm')
+                .send(obm)
+                .expect('Content-Type', /^application\/json/)
+                .expect(201)
+                .expect(function() {
+                    expect(nodesApi.putObmsByNodeId).to.have.been.calledWith('123',obm);
+                });
+        });
+
```

Comment:
At least miss two test cases:
(1) If `nodeId` is invalid, then GET will return 404.
(2) If `nodeId1 is invalid, then PUT will return 404.